sdk/__init__.py

```diff
@@ -23,8 +23,8 @@ class SDK:
 
     def __init__(
         self,
-        client_id: str,
-        client_secret: str,
+        client_id: str = "",
```

Comment:
I would vouch against turning these args as kwargs because that make them sound optional, which they are not

Reply:
Done